test/behaviour/concept/type/relationtype/RelationTypeSteps.java

```diff
@@ -111,6 +121,22 @@ public void relation_type_get_related_roles_do_not_contain(String relationLabel,
         for (Label label : roleLabels) assertFalse(actuals.contains(label));
     }
 
+    private Set<Label> relation_type_get_related_explicit_roles_actuals(String relationLabel) {
```

Comment:
we should normally put the private methods used in other methods below the callers

Reply:
mmm odd style but okay keep it for now